apps/sms/services/js/conversation/conversation_client.js

```diff
@@ -0,0 +1,79 @@
+/* global bridge,
+          streamClient,
+          Thread
+*/
+
+(function(exports) {
+  'use strict';
+
+  var debug = 0 ?
+    (arg1, ...args) => console.log('[ConversationClient] ' + arg1, ...args):
+    () => {};
+
+  /**
+   * Name of the service that is responsible for managing conversation.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'conversation-service';
+
+  /**
+   * Reference to active bridge client instance.
+   *
+   * @type {Client}
+   */
+  var client;
+
+  /**
+   * ConversationClient is a wrapper around bridge client connected to the
+   * bridge service hosted in a Shared Worker.
+   * @type {Object}
+   */
+  var ConversationClient = {
+    /**
+     * Initialized conversation service client bridge.
+     */
+    init() {
+      var serviceEndpoint = new SharedWorker(
+        '/services/js/conversation/conversation_service.js'
+      );
+
+      serviceEndpoint.port.start();
```

Comment:
yeah I agree it would be great if we can hide this part inside bridge lib. No matter what action you want to do with the worker(maybe disconnect/resume the connection ?), it should be accessible by bridge service API instead of access the worker directly.